build.gradle

```diff
@@ -21,7 +21,7 @@ apply plugin: 'edu.sc.seis.launch4j'
 apply plugin: "com.github.ben-manes.versions"
 
 sourceCompatibility = 1.8
-version = '0.5.2-1.8.9'
+version = '0.6.1'
```

Comment:
We need to discuss how to properly name this fork since it applies to versions other than 1.8.9, but this is not it. Something needs to follow the dash to differentiate it from https://github.com/FabricMC/fabric-installer.